src/lightning/pytorch/callbacks/spike.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
This probably needs to be added to the callbacks `__init__.py`

Reply:
done in https://github.com/Lightning-AI/lightning/pull/18014